src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1968,7 +1973,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         IDS_LogoLine1 = 56007
         IDS_LogoLine2 = 56008
         IDS_VBCHelp = 56009
-        IDS_InvalidPreprocessorConstantType = 56010
+        ' IDS_InvalidPreprocessorConstantType = 56010           Replaced by ERR_InvalidPreprocessorConstantType
```

Comment:
Discussed offline. Will mark as available as this value is not exposed externally to compiler consumers.

---
In reply to: [105975443](https://github.com/dotnet/roslyn/pull/17113#discussion_r105975443) [](ancestors = 105975443,105784631)